docs/index.rst

```diff
@@ -12,32 +12,32 @@ Configuration
    :caption: Configuration
 
 
-* Add the Crawlera middleware including it into the ``DOWNLOADER_MIDDLEWARES`` in your ``settings.py`` file::
+* Add the Zyte Smart Proxy Manager middleware including it into the ``DOWNLOADER_MIDDLEWARES`` in your ``settings.py`` file::
 
     DOWNLOADER_MIDDLEWARES = {
         ...
-        'scrapy_crawlera.CrawleraMiddleware': 610
+        'scrapy_zyte_proxy.SmartProxyManagerMiddleware': 610
     }
 
 * Then there are two ways to enable it
 
   * Through ``settings.py``::
 
-      CRAWLERA_ENABLED = True
-      CRAWLERA_APIKEY = 'apikey'
+      ZYTE_SPM_ENABLED = True
+      ZYTE_SPM_APIKEY = 'apikey'
```

Comment:
SPM needs to be discussed, the original plan was `ZYTE_PROXY`.

Reply:
I see that. Either we use Zyte Proxy or SPM consistently. Since decision was made to call it `zyte_proxy` we must avoid references to SPM here.

Reply:
Replaced.